### DISCUSSION TITLE
ASUS AX86U

### DISCUSSION THREAD

**Discussion by skyunix** [messense/aliyundrive-webdav#258] [Q&A] [answered]
armv8请问要下载哪个？

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/releases/download/v1.1.1/aliyundrivewebdav-merlin-arm384-aarch64.tar.gz

**Comment by longman-nj** [comment]
提交后，短时间提示：阿里云盘 进程未在运行！ Pls help
